app/views/users/privacy_settings.mobile.haml

```diff
@@ -0,0 +1,41 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+-#   licensed under the Affero General Public License version 3 or later.  See
+-#   the COPYRIGHT file.
+
+.stream_element
+  - content_for :page_title do
+    = t('.title')
+
+  = render 'shared/settings_nav'
+
+  .container-fluid
+    .col-lg-8.col-lg-offset-2
+      %h3
+        = t('.title')
+
+      = form_for current_user, url: user_path, html: { method: :put } do |f|
+        = f.error_messages
+
+        = f.fields_for :stream_preferences do |type|
+          #stream_prefs
+          = f.label :strip_exif, class: "checkbox" do
+            = f.check_box :strip_exif
+            = t('.strip_exif')
+
+          .small-horizontal-spacer
+          .clearfix
+            = f.submit t('users.edit.change'), class: 'btn btn-primary pull-right'
+
+      %hr
+      .small-horizontal-spacer
+      %h3
+        = t('.ignored_users')
+
+      - if @blocks.length.zero?
+        = t('.no_user_ignored_message')
+
+      - @blocks.each do |block|
+        = block.person_name
+        \-
+        = link_to t('.stop_ignoring'), block_path(block),
+          method: :delete
```

Comment:
I could use a partial to do that yeah. What's your opinion everybody? Is this view good enough like this for mobile or should it be improved?

![capture d ecran 2015-06-08 a 23 07 59](https://cloud.githubusercontent.com/assets/930064/8045373/3cbf9f50-0e33-11e5-92d3-505f04d1f3e2.png)